plugin/pulpcore/plugin/changeset/main.py

```diff
@@ -41,6 +42,9 @@ class ChangeSet:
         removals (SizedIterable): The content IDs to be removed.
         deferred (bool): Downloading is deferred.  When true, downloads are not performed
             but content is still created and added to the repository.
+        added (int): The number of content units successfully added.
+        removed (int): The number of content units successfully removed.
+        failed (int): The number of content units changes that failed.
```

Comment:
Suggest `The number of changes that failed.` because the failure could be a removal.